src/main/java/Event.java

```diff
@@ -0,0 +1,19 @@
+/**
+ * Represents a task that is an event.
+ * @author Aaron
+ */
+public class Event extends Task {
+    protected String from;
+    protected String to;
```

Comment:
you could consider changing 'from' to something like startDate, and 'to' to something like endDate, so that it is clearer and less ambiguous.